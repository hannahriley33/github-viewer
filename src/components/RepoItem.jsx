import React from 'react';
import PropTypes from 'prop-types';

const RepoItem = ({ repoName, url }) => (
  <a href={url}>{repoName}</a>
);

RepoItem.propTypes = {
  repoName: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired
};

export default RepoItem;
