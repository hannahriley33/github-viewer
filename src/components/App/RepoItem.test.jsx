import React from 'react';
import { shallow } from 'enzyme';
import RepoItem from '../RepoItem.jsx';

describe('RepoItem', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<RepoItem repoName="Rick-Sanchez-App"
      url="https://rickandmortyapi.com/api/character/avatar/1.jpeg" />);
    expect(wrapper).toMatchSnapshot();
  });
});
