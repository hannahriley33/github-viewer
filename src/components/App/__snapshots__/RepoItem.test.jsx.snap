// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoItem matches a snapshot 1`] = `
<a
  href="https://rickandmortyapi.com/api/character/avatar/1.jpeg"
>
  Rick-Sanchez-App
</a>
`;
